tools/metrics.py

```diff
@@ -0,0 +1,157 @@
+#! /usr/bin/env python3
```

Comment:
Make a new directory until `/tools` called `/planner_benchmarking` and place these in there with the 3 10/15/20% occupancy maps